### DISCUSSION TITLE
Hosting Nextjs app in IIS as sub Application under Website

### DISCUSSION THREAD

**Discussion by RichLearner** [vercel/next.js#57400] [Help]
### Summary

I am trying to deploy my nextJs app to IIS. I have been testing and trying out ways to deploy but I never seem to get the issue behind. The issue is that the page loads however I encountered some errors where the console says "GET {website_url} 404 Not Found"

This is what my web.config looks like

`
<configuration>
  <system.webServer>
    
    <webSocket enabled="false" />
    <handlers>
      
      <add name="iisnode" path="server.js" verb="*" modules="iisnode"/>
    </handlers>
    <rewrite>
      <rules>
        
        <rule name="NodeInspector" patternSyntax="ECMAScript" stopProcessing="true">
          <match url="^server.js\/debug[\/]?" />
        </rule>
        
        <rule name="StaticContent">
          <action type="Rewrite" url="public{REQUEST_URI}"/>
        </rule>
        
        <rule name="DynamicContent">
          <conditions>
            <add input="{REQUEST_FILENAME}" matchType="IsFile" negate="True"/>
          </conditions>
          <action type="Rewrite" url="server.js"/>
        </rule>
      </rules>
    </rewrite>
    
    <security>
      <requestFiltering>
        <hiddenSegments>
          <add segment="node_modules"/>
        </hiddenSegments>
      </requestFiltering>
    </security>
    
    <httpErrors existingResponse="PassThrough" />
    <iisnode node_env="production" />
    
    
  </system.webServer>
  <appSettings>
    <add key="API_URL" value="http://192.168.0.18:91/json/Default.aspx" />
    <add key="API_FOOD_URL" value="http://192.168.0.18:91/json/food_menu.aspx" />
    <add key="NEXT_PUBLIC_API_CORS_URL" value="http://192.168.0.18:91/json/Default.aspx" />
    <add key="NEXT_PUBLIC_API_FOOD_CORS_URL" value="http://192.168.0.18:91/json/food_menu.aspx" />
    <add key="NEXT_PUBLIC_API_URL" value="http://192.168.0.18:91/json/Default.aspx" />
    <add key="NEXT_PUBLIC_API_FOOD_URL" value="http://192.168.0.18:91/json/food_menu.aspx" />
  </appSettings>
</configuration>
`

### Additional information

```js
I have deployed 2 separate instances one which is an indivual website and one which is a sub application under a website.The one which I depployed individually is "next-web-order" which works totally fine. However the one hosted under ClubApp has an ASP.Net project and doesn't load like it should be. I'm guessing there is some problems regarding about my configuration as the build files are the same. Here is my web.config for the Nextjs project.
```

**Comment by ylberxhambazi** [comment]
Unverified, I don't know if this works?

https://docs.lextudio.com/blog/running-next-js-web-apps-on-iis-with-httpplatformhandler/
